api/db/changelog/db.changelog-29-admin-logging.xml

```diff
@@ -12,10 +12,16 @@ xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
       <column name="admin_user_id" type="bigint">
         <constraints nullable="false"/>
       </column>
-      <column name="target_id" type="bigint">
+      <column name="target_user_id" type="bigint" defaultValue="null">
+        <constraints nullable="true"/>
+      </column>
+      <column name="target_workspace_id" type="bigint" defaultValue="null">
+        <constraints nullable="true"/>
+      </column>
+      <column name="old_value_as_string" type="varchar(255)" defaultValue="">
```

Comment:
Will it be clear to a reader what the admin action actually was? Seems like I'll need to reverse-engineer this from the old_value/new_value, e.g. I'll just see REVOKED and need to know this refers to their data access while "false" refers to ID verification. I don't feel strongly on having old_value/new_value; I see this could be useful for programmatic use later, but maybe keep the `action` column you had before too (or rename, e.g. log/description)?

Reply:
good point -- I think I've been staring at this for too long...